packages/identity/src/logger.ts

```diff
@@ -1,3 +1,10 @@
-import { createQuietLogger } from '@quiet/logger'
+import { createQuietLogger, DEFAULT_INTERNAL_LOG_METHOD, __nodeConsoleLogger } from '@quiet/logger'
 
-export const createLogger = createQuietLogger('identity')
+export const createLogger = (() => {
```

Comment:
Everywhere that I'm doing this funky try/catch is meant to avoid the `fs` import errors on mobile.  The only place we load this package is in desktop environments and the backend so any of these common packages won't break in a react native environment.